src/test/resources/unzer.properties

```diff
@@ -1,3 +1,2 @@
-rest.endpoint=https://api.unzer.com
-rest.version=v1
+unzer.environment=SBX
```

Comment:
Might this file then also be obsolete? Unless you want to explicitly set SBX, which should be used by default anyways?.
But the valdiationUrls are also not used anymore, right?